config/templates/cloud-config-controller

```diff
@@ -344,7 +344,7 @@ write_files:
           - --tls-private-key-file=/etc/kubernetes/ssl/apiserver-key.pem
           - --client-ca-file=/etc/kubernetes/ssl/ca.pem
           - --service-account-key-file=/etc/kubernetes/ssl/apiserver-key.pem
-          - --runtime-config=extensions/v1beta1/networkpolicies=true
+          - --runtime-config=extensions/v1beta1/networkpolicies=true, batch/v2alpha1
```

Comment:
There is a space in the command argument, can you remove that?

I was going to ask to shield this behind an experimental flag like [this PR](https://github.com/coreos/kube-aws/pull/92/files) but i don't think enabling cronjobs is going to be dangerous..